ESSArch_Core/ip/utils.py

```diff
@@ -79,7 +79,8 @@ def generate_content_mets(ip):
     algorithm = ip.get_checksum_algorithm()
 
     generator = XMLGenerator()
-    generator.generate(files_to_create, folderToParse=ip.object_path, algorithm=algorithm)
+    for _ in generator.generate(files_to_create, folderToParse=ip.object_path, algorithm=algorithm):
```

Comment:
We can _probably_ use [`yield from`](https://docs.python.org/3/whatsnew/3.3.html#pep-380-syntax-for-delegating-to-a-subgenerator) here instead of the loop. (Note that the loop body also can be deleted, it's just that Github doesn't allow multi line suggestions atm)

```suggestion
    yield from generator.generate(files_to_create, folderToParse=ip.object_path, algorithm=algorithm)
```

Reply:
Very pythonic! :+1:  Me like! :)